specifyweb/frontend/js_src/lib/localization/locality.ts

```diff
@@ -292,4 +292,17 @@ export const localityText = createDictionary({
     'uk-ua': 'DD MM SS.SS N/S/E/W (32 45 42.84 N)',
     'de-ch': 'DD MM SS.SS N/S/O/W (32 45 42.84 N)',
   },
+  localityImportHeaderError: {
+    'en-us': 'Errors Found in Column Headers',
```

Comment:
(optional) is this better or worse?
```suggestion
    'en-us': 'Errors found in column headers',
```